addons/pos_viva_wallet/static/src/app/payment_viva_wallet.js

```diff
@@ -44,11 +44,13 @@ export class PaymentVivaWallet extends PaymentInterface {
         if (line) {
             line.set_payment_status("retry");
         }
-        this._show_error(
-            _t(
-                "Could not connect to the Odoo server, please check your internet connection and try again."
-            )
-        );
+        if (this.payment_method.viva_wallet_bearer_token !== "Bearer Token"){
```

Comment:
why do you need this condition ?

Reply:
This condition is required to validate wrong credentials and to show only credential related popup and not to show connectivity issue popup if credentials are wrong